universal/store/createStore.js

```diff
@@ -1,6 +1,9 @@
 import { applyMiddleware, compose, createStore } from 'redux'
 import thunk from 'redux-thunk'
 import makeRootReducer from './reducers'
+import createLogger from 'redux-logger'
+
+const logger = createLogger()
```

Comment:
@bbrock25 i went ahead and did that but it doesnt matter so long as the const is defined above where i call it in the scope....it doesn't get executed until it's used though, in this case in the push()

Am i missing something? Either way...it's in there now and changed to be defined within the __DEBUG__ block